src/Http/Client/Cookie/Cookie.php

```diff
@@ -0,0 +1,387 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.5.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Http\Client\Cookie;
+
+use Cake\Utility\Hash;
+use Cake\Utility\Security;
+use DateTimeInterface;
+use InvalidArgumentException;
+use RuntimeException;
+
+class Cookie implements CookieInterface
```

Comment:
Could this be `Cake\Http\Cookie`? Or is it specific to cookies that come out of the Http\Client?